week 10/testing.py

```diff
@@ -0,0 +1,63 @@
+"""
+CP1404/CP5632 Practical
+Testing code using assert and doctest
+"""
+
+import doctest
+from car import Car
+
+
+def repeat_string(s, n):
+    """Repeat string s, n times, with spaces in between."""
+    return " ".join([s] * n) # Fix the space
```

Comment:
Not in line with PEP 8: "Inline comments should be separated by at least two spaces from the statement"